test/Microsoft.AspNetCore.Rewrite.Tests/IISUrlRewrite/MiddleWareTests.cs

```diff
@@ -336,5 +336,35 @@ public class MiddlewareTests
 
             Assert.Equal(response.Headers.Location.OriginalString, "/foo");
         }
+
+        [Theory]
+        [InlineData("IsFile")]
+        [InlineData("isfile")]
+        [InlineData("IsDirectory")]
+        [InlineData("isdirectory")]
+        public async Task VerifyIsFileAndIsDirectoryParsing(string matchType)
```

Comment:
The same thing wouldn't happen. If it was parsed incorrectly and the default value of `Pattern `was set for the `matchType` then it would throw because no pattern is set in the conditions that use `IsFile` and `IsDirectory`. It is being parsed correctly 😃